### ISSUE TITLE
[TalkBack] After voice tone change, video cannot play. 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#237]
```
What steps will reproduce the problem?
Precondition : talkback on.
1. Setting->Language & input. 
2. Text-to-speed output
3. Voice tone change.
4. video play->video stop->video cannot play

What is the expected output? What do you see instead?
After voice tone change, video can play. 

What version of the product are you using? On what operating system?
android 4.2.1 talkback 3.3.*

Please provide any additional information below.


```

Original issue reported on code.google.com by `ezc...@gmail.com` on 22 Apr 2013 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
Cannot reproduce using YouTube video app. Please include Logcat output and 
reopen the bug if you are able to consistently reproduce this issue.
```

Original comment by `alanv@google.com` on 23 Apr 2013 at 2:23
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
attach logfile
```

Original comment by `ezc...@gmail.com` on 24 Apr 2013 at 12:01

Attachments:
- [bugreport.txt](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-237/comment-2/bugreport.txt)

**Comment by GoogleCodeExporter** [comment]
```
From the log, I believe this is an issue with the LG TTS. I am unable to 
reproduce using Google TTS.
```

Original comment by `alanv@google.com` on 24 Apr 2013 at 7:20
